.github/workflows/validate.yml

```diff
@@ -1,11 +1,5 @@
 name: validate
-on:
-  push:
-    branches:
-    - master
-  pull_request:
-    branches:
-    - '**'
+on: [push, pull_request]
```

Comment:
Please leave it as it is otherwise, CI would run every time we push any branch. @anfbermudezme